sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -459,7 +459,7 @@ case class UnresolvedStar(target: Option[Seq[String]]) extends Star with Unevalu
     }
   }
 
-  override def toString: String = target.map(_ + ".").getOrElse("") + "*"
+  override def toString: String = target.map(_.mkString("", ".", ".")).getOrElse("") + "*"
```

Comment:
Just wondering if this will be a kind of breaking change that some users use toString already to match plans, etc.


cc @cloud-fan

Reply:
this is already merged code. I hope they don't rely on this because moving from 2.12 to 2.13 would break their code.